js/src/css/abstracts/_variables.scss

```diff
@@ -6,6 +6,7 @@
 // Typography
 
 $gla-font-smallest: 11px;
+$gla-font-smaller: 12px;
```

Comment:
Similar to https://github.com/woocommerce/google-listings-and-ads/pull/1680#discussion_r980860508. Maybe the following style could also use this variable after resolving conflicts: https://github.com/woocommerce/google-listings-and-ads/blob/05c7f0fcf1cd3d072300026edd3133cc99cd3f56/js/src/attribute-mapping/index.scss#L11